content/components/only-manipulate-the-dom-via-the-renderer.md

```diff
@@ -0,0 +1,72 @@
+---
+title: only manipulate the DOM via the Renderer
+author: 
+  name: Billy Lando
+  url: https://github.com/billyjov
+---
+
+# Problem 
+
+According to the Angular documentation, relying on direct DOM access creates tight coupling between your application and rendering layers which will make it impossible to separate the two and deploy your application into a web worker. In addition, tight coupling restricts us from using **Ahead-of-Time** compilation which has obvious benefits, such as smaller bundle size or faster load times.
```

Comment:
> In addition, tight coupling restricts us from using **Ahead-of-Time** compilation which has obvious benefits, such as smaller bundle size or faster load times.

I'm not sure if this is correct. I have seen code before that was manually updating the dom using nativeElement but was still able to do a production build (so using AOT). I think it is more a problem for SSR and web worker but not really related to AOT.
Unless you can prove me wrong on this part? I'm 95% sure on this.

Reply:
Yes that might be true. I think I was confusing AoT with SSR here... but SSR should have it's own category IMO. So maybe we can just get rid of this small paragraph then. Good catch @KwintenP.

Reply:
@billyjov I would suggest to get rid of this sentence.

Reply:
Getting rid of it is best :) Thanks for testing!